torch/distributed/_tensor/device_mesh.py

```diff
@@ -167,9 +166,11 @@ def _get_or_create_default_group(self):
             # as an attribute of device mesh for future use. However, the detail is still TBD how we gonna use
             # this attribute, so we will implement this logic once we figure out the answer.
             self._seed = torch.cuda.initial_seed()
+        elif self.device_type == "ort":
```

Comment:
curious is `ort` both a device_type and also a communication backend? would `tensor.to("ort")` works?

Reply:
`ort` is a device type so `tensor.to("ort")` will work. but `ort` is not a communication backend.

Reply:
I think we don't need this special check anymore after https://github.com/pytorch/pytorch/pull/102336

Reply:
thanks, I've removed it